SPECS/annobin/annobin.spec

```diff
@@ -1,62 +1,151 @@
-Vendor:         Microsoft Corporation
-Distribution:   Azure Linux
-
 Name:    annobin
 Summary: Binary annotation plugin for GCC
-Version: 9.27
-Release: 4%{?dist}
-License: GPLv3+
-URL:     https://fedoraproject.org/wiki/Toolchain/Watermark
+Version: 12.49
+Release: 1%{?dist}
+License: GPL-3.0-or-later AND LGPL-2.0-or-later AND (GPL-2.0-or-later WITH GCC-exception-2.0) AND (LGPL-2.0-or-later WITH GCC-exception-2.0) AND GFDL-1.3-or-later
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+URL: https://sourceware.org/annobin/
```

Comment:
As part of extended -> core move could you also align the spacing for these tags? The spec linter should do that for you (just watch out, it may make too many changes, so it's good to run it on a clean commit).;)

Reply:
Fixed it for this and other files. Let me know if I missed any files